tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -125,16 +125,6 @@ public static void Initialize()
                 Log.Error(ex, ex.Message);
             }
 
-            try
-            {
-                var asm = typeof(Instrumentation).Assembly;
-                Log.Information($"[Assembly metadata] Location: {asm.Location}, CodeBase: {asm.CodeBase}, GAC: {asm.GlobalAssemblyCache}, HostContext: {asm.HostContext}, SecurityRuleSet: {asm.SecurityRuleSet}");
```

Comment:
Was this removed intentionally?

Reply:
Yes, I added this during functions work and I'm not sure it adds value. Also the GAC references fail in net5.0.